src/Process/TreeLayerProcessing.js

```diff
@@ -0,0 +1,10 @@
+export default function updateTreeLayer(processNodeFn, context, layer, node) {
+    if (!node) {
```

Comment:
With `initRootNodes` and `updateTreeLayer` as they are, itowns provide a framework to implement a tree-based layer: you just need to add these 2 properties and you're done.

If the `initRootNodes` must be done by the user, you transfer him responsabilities: the user must implement the func *and* decide when to call it (before or after scene creation ? before or after other layers creations ? etc etc)